compress/algorithm.go

```diff
@@ -0,0 +1,40 @@
+package compress
+
+import (
+	"bytes"
+	"fmt"
+	"os/exec"
+
+	"github.com/cdnjs/tools/util"
+)
+
+// Runs an algorithm with a set of arguments,
+// and returns its stdout as bytes.
+// Note, this function will panic if anything is
+// output to stderr.
+func runAlgorithm(alg string, args ...string) []byte {
+	cmd := exec.Command(alg, args...)
+	var stdOut, stdErr bytes.Buffer
+	cmd.Stdout, cmd.Stderr = &stdOut, &stdErr
+
+	err := cmd.Run()
+	util.Check(err)
+
+	if stdErr.Len() > 0 {
+		panic(fmt.Sprintf("%s failed: %s", alg, stdErr.String()))
+	}
+
+	return stdOut.Bytes()
+}
+
+// Brotli11 returns a brotli compressed file as bytes
+// at optimal compression (quality 11).
+func Brotli11(filePath string) []byte {
+	return runAlgorithm("brotli", "-c", "-q", "11", filePath)
```

Comment:
Yet another package which need to be installed. Should we document it somewhere, maybe in the README?

/ref #57

Reply:
good point, will update README, thanks 😄